libafl/src/stages/pruning.rs

```diff
@@ -1,194 +0,0 @@
-//! Corpus pruning stage
```

Comment:
this one currently doesn't work. i have a working one in my repo. i'll commit after the experiment on fuzzbench